core/src/main/java/com/yahoo/ycsb/workloads/CoreWorkload.java

```diff
@@ -393,8 +416,18 @@ public void init(Properties p) throws WorkloadException {
     String scanlengthdistrib =
         p.getProperty(SCAN_LENGTH_DISTRIBUTION_PROPERTY, SCAN_LENGTH_DISTRIBUTION_PROPERTY_DEFAULT);
 
-    int insertstart =
-        Integer.parseInt(p.getProperty(INSERT_START_PROPERTY, INSERT_START_PROPERTY_DEFAULT));
+  int insertstart =
+    Integer.parseInt(p.getProperty(INSERT_START_PROPERTY,INSERT_START_PROPERTY_DEFAULT));
+  int insertcount =
+    Integer.parseInt(p.getProperty(INSERT_COUNT_PROPERTY,String.valueOf(recordcount-insertstart)));
```

Comment:
You have no space after the ',' on this line.... whereas in rest of code base, there is always a space after a ','

See also the '-' sign... how there are not spaces around it but there are when you look at operators in the background code